packages/proof/src/index.ts

```diff
@@ -1,7 +1,6 @@
 import { packGroth16Proof, unpackGroth16Proof } from "@zk-kit/utils"
 import generateProof from "./generate-proof"
-import getSnarkArtifacts from "./get-snark-artifacts.node"
 import verifyProof from "./verify-proof"
 
 export * from "./types"
-export { generateProof, getSnarkArtifacts, packGroth16Proof, unpackGroth16Proof, verifyProof }
```

Comment:
It would be nice to have `getSnarkArtifacts` for node as a utility function to make it easier for devs to download the artifacts in case they need it because the automatic download does not work for them.

Reply:
If the automatic download doesn't work, that function won't work used externally either I think :/

I think it can actually be useful if you can define your own URLs.

Reply:
Yes, true. It should be removed from here. Maybe adding this feature somewhere else to do it.

Reply:
That can be part of another issue. It could be worth checking the code again and see if some other functions can be moved to the utils package.